### ISSUE TITLE
tar: bash-4.3.046-1/depends: Not found in archive

### ISSUE THREAD

**Issue by mildred** [drinkcat/chroagh#122]
When I try to install a ArchLinux chroot using the command in the README, I get the following error:

```
tar: bash-4.3.046-1/depends: Not found in archive
```

The full log:

```
$ sudo sh -xe installer/main^Ch -r arch -t xfce
WARNING: USB booting is enabled; consider disabling it.
WARNING: Signed boot verification is disabled; consider enabling it.
You can use the following command: sudo crossystem dev_boot_usb=0 dev_boot_signed_only=1
Installing arch-x86_64 chroot to /usr/local/chroots/arch
Fetching repository packages list...
Fetching core...
######################################################################## 100.0%
Fetching community...
######################################################################## 100.0%
Fetching extra...
######################################################################## 100.0%
Downloading and extracting packages...
Downloading and installing bash...
######################################################################## 100.0%
tar: bash-4.3.046-1/depends: Not found in archive
tar: Exiting with failure status due to previous errors
tar: bash-4.3.046-1/depends: Not found in archive
tar: Exiting with failure status due to previous errors
Downloading and installing gawk...
######################################################################## 100.0%
Downloading and installing grep...
######################################################################## 100.0%
tar: .BUILDINFO: Cannot open: File exists
tar: Exiting with failure status due to previous errors
```

The issue is that the `bash-4.3.046-1/depends` file does not exists in the `code.db` database (a tar archive).

**Comment by mildred** [comment]
It seems some packages do not have a depends file. Perhaps this is because there is no dependency. The `getdbfield` function in `installer/arch/bootstrap` should perhaps be modified for that specific case

**Comment by mildred** [comment]
This is the same as the problem fixed in #107 

You should use https://github.com/elkangaroo/chroagh which works way better
